src/NuGet.Core/NuGet.Packaging/Core/PackageDependencyInfo.cs

```diff
@@ -47,17 +49,14 @@ public PackageDependencyInfo(string id, NuGetVersion version, IEnumerable<Packag
         /// <summary>
         /// Package dependencies
         /// </summary>
-        public IEnumerable<PackageDependency> Dependencies
-        {
-            get { return _dependencies; }
-        }
+        public IEnumerable<PackageDependency> Dependencies => _dependencies;
```

Comment:
nit: This can just be a property correct? Or is there somewhere in the class where we use the fact that `_dependencies` is an array/

Reply:
Made it into an auto property.